defaults/config.js

```diff
@@ -233,6 +233,17 @@ module.exports = {
 		//
 		tls: true,
 
+		//
+		// Enable certificate verification
+		//
+		// If true, the server certificate is verified against
+		// the list of supplied CAs by your node.js installation.
+		//
+		// @type     boolean
+		// @default  true
+		//
+		rejectUnauthorized: true,
```

Comment:
It's kind of a headache going through the different combinations of our settings (I've rewritten this comment 3 times already), but ideally the checkbox would change to enabled whenever user is changing the settings to a different network.
That's an amount of extra work probably outside the scope of this PR though.

Reply:
I'm not sure what you mean by

> the checkbox would change to enabled whenever user is changing the settings to a different network.

Reply:
Yeah, I was wondering if that was clear...
What I meant was that as soon as the user starts typing something that is not the default network, we should probably set the checkbox to the highest security setting. If settings value is `false`, then by default the UI suggests the least secure option.

But as mentioned, this is likely more work than it's worth, and I can think of a couple edge cases making this painful.

Reply:
I think I see what you mean. This feels like a lot of effort for little gain there (what we really need is network editing!).